### PR TITLE
New command for getting duplicate convert status 

### PR DESCRIPTION
**Pull request by ko101**
New feature for getting duplicate split/move completion status .
Command name : duplicate-convert-status

### CODE REVIEW

**Review by allmightyspiff — APPROVED** [review]
Looks good to me, when the API for this is in production, I will merge this in.
Please comment on this pull request when that is officially available.

File: SoftLayer/managers/storage.py [hunk 1]
```diff
@@ -577,3 +577,12 @@ def convert_dep_dupe(self, volume_id):
         return self.client.call('Network_Storage',
                                 'convertCloneDependentToIndependent',
                                 id=volume_id)
+
+    def split_percentage(self, volume_id):
+        """Get the Clone split/move status completion of a duplicate volume
```
Comment (caberos) [inline SoftLayer/managers/storage.py:582]: please put the link reference

File: SoftLayer/CLI/file/duplicate_convert_status.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand,
+               epilog="""Get status for split or move completed percentage of a given file duplicate volume.""")
+@click.argument('volume-id')
+@environment.pass_env
+def cli(env, volume_id):
+    """Get status for split or move completed percentage of a given file duplicate volume."""
+    table = formatting.Table(['username', 'active_conversion_start_time', 'completed_percentage'])
```
Comment (caberos) [inline SoftLayer/CLI/file/duplicate_convert_status.py:16]: uppercase

File: SoftLayer/CLI/file/duplicate_convert_status.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    file_manager = SoftLayer.FileStorageManager(env.client)
+
+    value = file_manager.split_percentage(
```
Comment (caberos) [inline SoftLayer/CLI/file/duplicate_convert_status.py:20]:
```suggestion
   value = file_manager.split_percentage(volume_id)
```

File: SoftLayer/CLI/block/duplicate_convert_status.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+               epilog="""Get status for split or move completed percentage of a given block duplicate volume.""")
+@click.argument('volume-id')
+@environment.pass_env
+def cli(env, volume_id):
+    """Get status for split or move completed percentage of a given block duplicate volume."""
+    table = formatting.Table(['username', 'active_conversion_start_time', 'completed_percentage'])
+
+    block_manager = SoftLayer.BlockStorageManager(env.client)
+
+    value = block_manager.split_percentage(
```
Comment (caberos) [inline SoftLayer/CLI/block/duplicate_convert_status.py:20]:
```suggestion
    value = block_manager.split_percentage(volume_id)
```

**Review by caberos — CHANGES_REQUESTED** [review]

File: SoftLayer/managers/storage.py (hunk 1 above)
Comment (ko101) [reply SoftLayer/managers/storage.py:582]: ahh.. which link reference to you are asking here...jira for API that's consuming this - https://jira.softlayer.local/browse/FBLOCK-4520

File: SoftLayer/CLI/file/duplicate_convert_status.py (hunk 2 above)
Comment (ko101) [reply SoftLayer/CLI/file/duplicate_convert_status.py]: Done

[the remark this answers is not in the document]
Comment (ko101) [reply SoftLayer/CLI/file/duplicate_convert_status.py]: Done

File: SoftLayer/CLI/block/duplicate_convert_status.py (hunk 4 above)
Comment (ko101) [reply SoftLayer/CLI/block/duplicate_convert_status.py]: Done

File: SoftLayer/CLI/file/duplicate_convert_status.py (hunk 3 above)
Comment (ko101) [reply SoftLayer/CLI/file/duplicate_convert_status.py]: Done

**Comment by ko101** [comment]
Hi @allmightyspiff , the api has released yesterday IST, Requesting you to please merge the this PR.
Thankyou

**Pull request merged**